commands/retry.js

```diff
@@ -2,6 +2,7 @@
 
 let cli = require('heroku-cli-util')
 let co = require('co')
+let output = require('heroku-apps/lib/output')
```

Comment:
frankly I think it would be better just to duplicate the code here. heroku-apps is a huge depedency and there is no guarantee that the code might move around or be changed in a patch release. I think it's useful enough to share within a repo but not so useful to have a new package for it. That's definitely my opinion though as someone that doesn't use things like `left-pad` at all.